sites/default/conf/core.extension.yml

```diff
@@ -32,6 +31,7 @@ module:
   field_ui: 0
   file: 0
   filter: 0
+  google_analytics: 0
```

Comment:
This should be disabled as we don't want to track, at least with google analytics.

Reply:
Done.